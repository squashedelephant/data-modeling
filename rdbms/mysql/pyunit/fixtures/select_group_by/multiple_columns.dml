INSERT INTO forecast(year, division, quarter, sales) VALUES(2017, 'US East', 'q1', 60000.00);
INSERT INTO forecast(year, division, quarter, sales) VALUES(2017, 'US East', 'q2', 70000.00);
INSERT INTO forecast(year, division, quarter, sales) VALUES(2017, 'US East', 'q3', 70000.00);
INSERT INTO forecast(year, division, quarter, sales) VALUES(2017, 'US East', 'q4', 80000.00);
INSERT INTO forecast(year, division, quarter, sales) VALUES(2017, 'US Midwest', 'q1', 50000.00);
INSERT INTO forecast(year, division, quarter, sales) VALUES(2017, 'US Midwest', 'q2', 40000.00);
INSERT INTO forecast(year, division, quarter, sales) VALUES(2017, 'US Midwest', 'q3', 40000.00);
INSERT INTO forecast(year, division, quarter, sales) VALUES(2017, 'US Midwest', 'q4', 50000.00);
INSERT INTO forecast(year, division, quarter, sales) VALUES(2017, 'US West', 'q1', 80000.00);
INSERT INTO forecast(year, division, quarter, sales) VALUES(2017, 'US West', 'q2', 90000.00);
INSERT INTO forecast(year, division, quarter, sales) VALUES(2017, 'US West', 'q3', 100000.00);
INSERT INTO forecast(year, division, quarter, sales) VALUES(2017, 'US West', 'q4', 125000.00);
