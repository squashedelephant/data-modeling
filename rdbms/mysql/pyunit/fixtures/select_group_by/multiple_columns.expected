2017L
US East
280000.00
2017L
US Midwest
180000.00
2017L
US West
395000.00
